client/app/common/less/common.less

```diff
@@ -94,3 +94,7 @@ button.telecom-link {
 .dropdown-menu li:before {
     content: none;
 }
+
+label.oui-switch.oui-switch-toggle {
+    margin: 0px 10px -7px 10px !important;
```

Comment:
Isn't it supposed to be part of oui-kit?

Reply:
small css helper to use oui-switch as a toggle button with labels on each sides
it will probably will be removed later when oui-kit implements toggle switches